includes/cli-configuration-artifacts-path.md

```diff
@@ -0,0 +1,9 @@
+---
+author: tdykstra
+ms.author: tdykstra
+ms.date: 26/04/2024
+ms.topic: include
+---
+- **`--artifacts-path <ARTIFACTS_DIR>`**
+
+  All build output files from the executed command will go in subfolders under the specified path, separated by project. For more information see [Artifacts Output Layout](../docs/core/sdk/artifacts-output.md). Available since .NET 8.+
```

Comment:
```suggestion
  All build output files from the executed command will go in subfolders under the specified path, separated by project. For more information see [Artifacts Output Layout](../docs/core/sdk/artifacts-output.md). Available since .NET 8 SDK.
```